src/components/Form/DeleteDocumentForm.jsx

```diff
@@ -65,7 +65,7 @@ const DeleteDocumentForm = () => {
       <Box display="flex" justifyContent="center" sx={{ minWidth: 120 }}>
         <form onSubmit={handleDeleteDocument}>
           <InputLabel id="delete-doctype">
-            <em>Select Document Type to Delete:</em>
+            <em>Select Document Type:</em>
```

Comment:
I think we can simplify these further with the colons in "Select Document Type:"

Since they'll just going to be labels for the input fields in MUI, think we can have "Select Document Type" as the final form